src/components/SharingIcon/SharingIcon.styl

```diff
@@ -3,7 +3,7 @@ purple=#a75bcb
 green=#35ce68
 
 .sharing-icon
-    background-image embedurl('../assets/icons/ui/icon-share.svg')
+    background-image embedurl('../assets/icons/ui/share.svg')
```

Comment:
This is on cozy-ui I thinks the path is not good?

Reply:
Why ? The stylus directory of cozy-ui is added to the stylus path.

+ stylus
+ assets

So you go up one directory and then you're good.

Reply:
Ca n'a pas fonctionné le changement, je pense qu'avec le ../assets/icons on le force à aller regarder dans cozy-ui.

```
  Module not found: Error: Can't resolve 'icons/ui/share.svg' in '/Users/pbrowne/code/cozy/banks/src/components/SharingIcon'
```